### PR TITLE
Create listing for population + demographics data

### PR DESCRIPTION
**Pull request by migurski**
High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/dataforgood-fb-hrsl.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Name: High Resolution Population Density Maps + Demographic Estimates by CIESIN and Facebook
+Description: |
+  The population of the world, allocated to 1 arcsecond blocks. This
+  refines CIESIN’s Gridded Population of the World project, using
+  machine learning models on high-resolution worldwide Digital Globe
+  satellite imagery.
+Documentation: https://dataforgood.fb.com/tools/population-density-maps/
+Contact: disastermaps@fb.com
```
Comment (jflasher) [inline datasets/dataforgood-fb-hrsl.yaml]: Can you add a ManagedBy Field like `[Facebook](https://dataforgood.fb.com/)` or `[Facebook](https://facebook.com)`?

File: datasets/dataforgood-fb-hrsl.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+UpdateFrequency: Quarterly
+Tags:
```
Comment (jflasher) [inline datasets/dataforgood-fb-hrsl.yaml]: Based on our current tags (we don't tag by country or company name), I'd recommend the below list.

 - population
  - demographics
  - machine learning
  - aerial imagery
  - satellite imagery
  - image processing
  - geospatial
  - sustainability
  - disaster response
  - aws-pds

I'll add population and demographics to the list of allowed tags now.

File: datasets/dataforgood-fb-hrsl.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  - machine learning
+  - aerial imagery
+  - satellite imagery
+  - image processing
+  - geospatial
+  - CIESIN
+  - Facebook
+  - DigitalGlobe
+  - Columbia
+License: http://www.gdeltproject.org/about.html#termsofuse
```
Comment (jflasher) [inline datasets/dataforgood-fb-hrsl.yaml]: Just confirming this is the license you'd like to use?

Comment (migurski) [reply datasets/dataforgood-fb-hrsl.yaml]: Thanks, I think I left this unmodified from a copy-paste. Updated with info from https://www.ciesin.columbia.edu/data/hrsl/

**Comment by migurski** [comment]
I rebased to fix the tag-related build errors, this should be good to go.

**Comment by jflasher** [comment]
Just a note that if the citation looks good, we should use https://github.com/awslabs/open-data-registry/pull/309 and close this one.

**Pull request closed without merging**